tests/msg_send_receive/tests/01-run.py

```diff
@@ -1,33 +1,19 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Wait. Didn't we decide that setting it this way was a bad idea. Something rings a bell here. @OlegHahm, @cgundogan, @LudwigKnuepfer, @Kijewski do you remember?